mapie/conformity_scores/conformity_scores.py

```diff
@@ -243,13 +243,16 @@ def get_quantile(
         NDArray of shape (1, n_alpha) or (n_samples, n_alpha)
             The quantile of the conformity scores.
         """
+        n_ref = conformity_scores.shape[-1]
         quantile = np.column_stack([
             np_nanquantile(
                 conformity_scores.astype(float),
                 _alpha,
                 axis=axis,
                 method=method
-            )
+            ) if 0 < _alpha < 1
```

Comment:
I find this code a bit convoluted. Would it be possible to write it in a simpler way?

Reply:
I agree, but everything is stacked with the method `np.column_stack`. So it's hard to make things simpler, at the risk of writing too many lines to manage the if/else condition outside the method.